src/js/RPE.ts

```diff
@@ -0,0 +1,159 @@
+
+/**
+ *  "EffortLevel"
+ *
+ *  The Rated Perceived Exertion scale is an attempt to quantify a 1-rep max weight given an overall "level of effort" and
+ *  "number of reps". Each level of effort bears a different meaning:
+ *
+ *   6 = Hard
+ *     .  .
+ *    \____/
+ *
+ *   7 = Harder
+ *     *  *'
+ *    \____/
+ *
+ *   8 = Very Hard
+ *     *  *'
+ *     ____
+ *
+ *   9 = Extremely hard
+ *     *  *'
+ *      O
+ *
+ *   10 = Maximum Effort
+ *     X  X'
+ *      O
+ *
+ *  */
+export type EffortLevel = "10" | "9" | "8" | "7" | "6";
```

Comment:
Having never written ts before, curious why these are strings instead of integers? Same with `RepCount`.

Reply:
They could certainly be numbers. I left them as strings for backwards compatibility. Since they're just placeholders for things and the relative difficulty factors aren't computed mathrmatically from them it doesn't change much if they're numbers. wdyt?